# HG changeset patch
# User tabbott@mit.edu
# Date 1217648612 14400
# Node ID dd50e2fe8e5f5238e156ba36b89bcaeef398bb01
# Parent  c19e40e79fcfecf32fc8793b83bd4b445d33692e
Fix missing #! lines in the extcode spkg.

diff -r c19e40e79fcf -r dd50e2fe8e5f mirror
--- a/mirror	Fri Jul 11 11:31:32 2008 -0700
+++ b/mirror	Fri Aug 01 23:43:32 2008 -0400
@@ -1,4 +1,4 @@
-
+#!/bin/sh
 cd $HOME/sage/web/dist/src/extcode-darcs/
 darcs changes > changelog_darcs.txt
 echo "uploading"
diff -r c19e40e79fcf -r dd50e2fe8e5f pari/dokchitser/testall
--- a/pari/dokchitser/testall	Fri Jul 11 11:31:32 2008 -0700
+++ b/pari/dokchitser/testall	Fri Aug 01 23:43:32 2008 -0400
@@ -1,3 +1,4 @@
+#!/bin/sh
 echo "\\r ex-bsw" | sage -gp
 echo "\\r ex-chgen" | sage -gp
 echo "\\r ex-chqua" | sage -gp
diff -r c19e40e79fcf -r dd50e2fe8e5f spkg-dist
--- a/spkg-dist	Fri Jul 11 11:31:32 2008 -0700
+++ b/spkg-dist	Fri Aug 01 23:43:32 2008 -0400
@@ -1,3 +1,4 @@
+#!/bin/sh
 if [ $# -ne 2 ]; then
     echo "Usage: $0 <SAGE_VERSION> <SAGE_ROOT>"
     echo "Build SAGE package in the extcode directory."
